[docs] remove the installation linking step from the GitHub builds guide and optimize images (#27732)
expo/expo: docs/pages/build/building-from-github.mdx

Install the Expo GitHub App on your GitHub account.

> > **Note:** You must be an admin of the Expo account to install the app.

Reviewer: ```suggestion
> **Note:** You must have [Owner or Admin access](/accounts/working-together/#manage-access) of the Expo account to install the app.
```

Let's link to Owner and Admin role and also specify Owner and Admin both so as not leave any room for confusion.

<ImageSpotlight
  alt="The GitHub app installation UI"
  src="/static/images/eas-build/build-from-github/install-github-app.png"
/>

> Then, you'll be able to link your repository to your Expo project.

Reviewer: ```suggestion
Then, link the GitHub repository to your Expo project.
```

> > **Note:** You can only link GitHub organization repositoriess to Expo organizations.

Reviewer: ```suggestion
> **Note:** You can only link [GitHub organization repositories](https://docs.github.com/en/organizations) to Expo organizations.
```

- Add a link to GitHub organization repo's
- Fix typo

<ImageSpotlight
  alt="The repository selector on the Expo project GitHub settings page"

>   src="/static/images/eas-build/build-from-github/connect-a-repository.png"

Reviewer: ```suggestion
  src="/static/images/eas-build/build-from-github/connect-a-repository.png"
  style={{ width: 600 }}
```
The image rendered here is a bit large. Let's add a width.

/>

> If you want to add a repository from a different GitHub account, click the "Add new account" option

Reviewer: ```suggestion
To add a repository from a different GitHub account, click the **Add new account** option in the account selector dropdown.
```

- Let's be concise here
- expo.dev settings are bold in our docs

> in the account selector dropdown.

Reviewer: ```suggestion
```

Added this in the above LOC by applying Prettier formatting.

<ImageSpotlight
  alt="The account selector on the Expo project GitHub settings page"

>   src="/static/images/eas-build/build-from-github/add-new-account.png"

Reviewer: ```suggestion
  src="/static/images/eas-build/build-from-github/add-new-account.png"
  style={{ width: 400 }}
```

Same as above.
